ballerina/Dependencies.toml

```diff
@@ -5,12 +5,12 @@
 
 [ballerina]
 dependencies-toml-version = "2"
-distribution-version = "2201.11.0-20241112-214900-6b80ab87"
+distribution-version = "2201.11.0-20241121-075100-c4c87cbc"
 
 [[package]]
 org = "ballerina"
 name = "crypto"
-version = "2.7.3"
+version = "2.7.2"
```

Comment:
Can you try deleting the Dependencies.toml file and build again. Here the crypto version should be `2.8.0`.

Reply:
Done